### PR TITLE
Added metadata for PyPI and bumped version

### PR DESCRIPTION
**Pull request by shimizust**
## Summary
- Added metadata to populate the PyPI entry
- bumped version

## Testing Done
`twine check dist/*`
```
Checking dist/liger_kernel-0.1.1-py3-none-any.whl: PASSED
Checking dist/liger_kernel-0.1.1.tar.gz: PASSED
```

`python -m readme_renderer README.md --output rendered_readme.html` --> renders properly

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+    description="Efficient Triton kernels for LLM Training",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    license="BSD-2-Clause",
+    url="https://github.com/linkedin/Liger-Kernel",
     package_dir={"": "src"},
     packages=find_namespace_packages(where="src"),
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
```
Comment (ByronHsu) [inline setup.py:17]: where do we get these?

File: setup.py [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+        'Intended Audience :: Education',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Topic :: Software Development :: Libraries',
+        'Topic :: Scientific/Engineering :: Artificial Intelligence',
+    ],
+    keywords='triton kernels LLM training deep learning Hugging Face PyTorch GPU optimization',
```
Comment (ByronHsu) [inline setup.py:28]: are they separated by space?

Comment (shimizust) [reply setup.py]: good call, I think it is comma separated

File: setup.py (hunk 1 above)
Comment (shimizust) [reply setup.py:17]: From here: https://pypi.org/classifiers/

**Review by ByronHsu — APPROVED** [review]

**Pull request merged**